Sage/carbon
# fix(dialog-full-screen): update contentRef prop to be optional in type definition FE-5162

fix #5121

### Proposed behaviour


Updates the `contentRef` prop to be optional in `dialog-full-screen.d.ts` file

### Current behaviour


The `contentRef` prop is required in `dialog-full-screen.d.ts` file

<img width="506" alt="image" src="https://user-images.githubusercontent.com/44157880/167164500-f9aa3fe6-53bb-4230-a592-14b64e57bfba.png">

### Checklist




#### QA


### Additional context



### Testing instructions



The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.


https://codesandbox.io/s/wispy-currying-0ok47e -- will need to be exported and run locally
